whois-commons/src/main/java/net/ripe/db/whois/common/dao/jdbc/JdbcRpslObjectDao.java

```diff
@@ -207,6 +207,7 @@ public RpslObject getByKeyOrNull(final ObjectType type, final CIString key) {
             try {
                 result.add(getByKey(type, searchKey));
             } catch (EmptyResultDataAccessException ignored) {
+                LOGGER.debug("EmptyResultDataAccessException {} in {} class ", ignored.getMessage(), ignored.getClass().getName());
```

Comment:
This is an expected exception, if there is no matching row in the database, can we ignore it instead?